packages/manager/apps/web/client/app/dns-zone/zone.routing.js

```diff
@@ -23,6 +21,52 @@ export default /* @ngInject */ ($stateProvider) => {
       defaultFilterColumn: () => 'name',
       header: /* @ngInject */ ($translate) => $translate.instant('zones_title'),
       customizableColumns: () => true,
+      columns: /* @ngInject */ ($translate) => [
+        {
+          property: 'name',
+          title: $translate.instant('zones_domain_name'),
+          template: `<a data-ng-href="{{ $ctrl.getServiceNameLink($row) }}" data-ng-bind="$row.name"></a>`,
+          searchable: true,
+          sortable: 'asc',
+          filterable: true,
+        },
+        {
+          property: 'hasDnsAnycast',
+          title: 'DNS Anycast',
+          template: `<span data-ng-bind="$row.hasDnsAnycast ? '${$translate.instant(
+            'zones_enabled',
+          )}' : '${$translate.instant(
+            'zones_disabled',
+          )}'" class="oui-badge" ng-class="{'oui-badge_success': $row.hasDnsAnycast, 'oui-badge_error': !$row.hasDnsAnycast}"></span>
+          `,
```

Comment:
```suggestion
          template: `<span data-ng-bind="'${$translate.instant($row.hasDnsAnycast ? 'zones_enabled' : 'zones_disabled')}'" class="oui-badge" ng-class="{'oui-badge_success': $row.hasDnsAnycast, 'oui-badge_error': !$row.hasDnsAnycast}"></span>
          `,
```
I think this is easier to read, what do you think ?

Reply:
Hello @JacquesLarique ,
It seems easier but I can't get the code to work with this method or via translate as below, but I'd have to change the rest to keep it consistent:

```js
template: `<span data-ng-bind="'{{ $row.hasDnsAnycast ? 'zones_enabled' : 'zones_disabled' | translate }}'" class="oui-badge" ng-class="{'oui-badge_success': $row.hasDnsAnycast, 'oui-badge_error': !$row.hasDnsAnycast}"></span>`

```